src/main/scala/mesosphere/marathon/core/task/tracker/impl/TaskSerializer.scala

```diff
@@ -122,18 +114,12 @@ object TaskSerializer {
     def setReservation(reservation: Task.Reservation): Unit = {
       builder.setReservation(ReservationSerializer.toProto(reservation))
     }
-    def setLaunched(appVersion: Timestamp, status: Task.Status, networking: Task.Networking): Unit = {
+    def setLaunched(appVersion: Timestamp, status: Task.Status, ports: Seq[Int]): Unit = {
       builder.setVersion(appVersion.toString)
       builder.setStagedAt(status.stagedAt.toDateTime.getMillis)
       status.startedAt.foreach(startedAt => builder.setStartedAt(startedAt.toDateTime.getMillis))
       status.mesosStatus.foreach(status => builder.setStatus(status))
-      networking match {
-        case Task.HostPorts(hostPorts) =>
-          builder.addAllPorts(hostPorts.view.map(Integer.valueOf(_)).asJava)
-        case Task.NetworkInfoList(networkInfoList) =>
-          builder.addAllNetworks(networkInfoList.asJava)
```

Comment:
I would love a comment in the marathon.proto which explains that the networks info in the protocol buffers is redundant and we got rid of that. Maybe even rename the field to "DEPRECATED_..." or comment it out.